Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -396,6 +397,8 @@ Button/Hotkey/AP_TimeWarp=C
 Button/Hotkey/AP_TorrasqueChrysalis=Q
 Button/Hotkey/AP_TrainHERC=H
 Button/Hotkey/AP_TransportMode=T
+Button/Hotkey/AP_TyrannozorBarrageOfSpikes=Q
+Button/Hotkey/AP_TyrannozorMerge=G

```

Comment:
I'd pick `T` as the hotkey (`T` for Tyrannozor)

Reply:
For some reason I thought T was a hotkey for a default action, like patrol, and just didn't check.  I've updated it.